src/openscad.cc

```diff
@@ -1196,3 +1196,22 @@ int main(int argc, char **argv)
 
   return rc;
 }
+
+#ifdef _WIN32
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wdeprecated-declarations"
+static std::wstring_convert<std::codecvt_utf8_utf16<wchar_t>, wchar_t> convert_wchar;
+#pragma GCC diagnostic pop
+std::string wchar_to_string(wchar_t *wcp) { return convert_wchar.to_bytes(wcp); }
+
+int wmain(int argc, wchar_t **argv)
+{
+  char *argv8[argc + 1];
+  for (int i = 0; i < argc; i++) {
+    argv8[i] = strdup(wchar_to_string(argv[i]).c_str());
+  }
+  argv8[argc] = NULL;
+
+  return (main(argc, argv8));
```

Comment:
Should we free the strdup'ed memory after main() returns?

Reply:
Does anything interesting happen after main() returns?

Reply:
Generally, I'm happy never deallocating memory that's intended to be kept until the end of the program.   Deallocating during shutdown seems like complexity and an opportunity for bugs, for no real value.

Reply:
Just to silence memory leak checkers

Reply:
Hmm.  At work, we only run memory leak checkers on core dumps (perhaps dumped from running processes).  I don't think we even have a simple way to run one at exit time.

I bet that there are lots of places throughout libraries that leave memory allocated at exit time, but indeed this is novel in that there would be no remaining pointers to the allocated memory.  Sold, added, pushed.

Hmm:  I didn't check for the `strdup()` failing.  I could, but if `strdup()` is failing on startup we have far worse problems.  I can add checks if you like.  (Actually, if `strdup()` fails anywhere we have serious problems.  My heretical opinion is that nothing should ever check for memory allocation failures, because in practice they are not survivable.)

Reply:
We generally don't check for memory allocation errors. In C++ this usually boils down to someone throwing std::bad_alloc, but I believe we just rely on someone catching that and crashing appropriately. In your case, it will just result in a segmentation fault whenever someone accesses the data. Crash may be harder to find, but it's unlikely to happen.